package.json

```diff
@@ -92,37 +92,158 @@
         "editor.trimAutoWhitespace": true
       },
       "editor.tokenColorCustomizations": {
-        "[*Dark*]": {
+        "[*Light*]": {
           "textMateRules": [
             {
-              "scope": "comment",
+              "scope": "comments",
```

Comment:
I think we expect comment from the semantic highlighter.
Now I see that only textmate token is there for comments when I expect Juvix file